### PR TITLE
Serialize manifest imports for mapped handles that are owned by manifests.

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

**Comment by shans** [comment]
Should fix the remaining part of #822

File: runtime/arc.js [hunk 1]
```diff
@@ -127,6 +127,14 @@ class Arc {
 
     let id = 0;
     let schemaSet = new Set();
+    let importSet = new Set();
+    for (let handle of this._activeRecipe.views) {
+      if (handle.fate == 'map')
+        importSet.add(this.context.findManifestUrlForHandleId(handle.id));
+    }
+    for (let url of importSet.values())
+      resources += `import '${url}\n`;
```
Comment (dstockwell) [inline runtime/arc.js]: missing a `'`

File: runtime/manifest.js [hunk 2]
```diff
@@ -188,6 +191,9 @@ class Manifest {
   findHandleById(id) {
     return this._find(manifest => manifest._handles.find(handle => handle.id == id));
   }
+  findManifestUrlForHandleId(id) {
+    return this._find(manifest => manifest._handleManifestUrls.get(id));
```
Comment (dstockwell) [inline runtime/manifest.js]: I think we could avoid needing the map with something like:
```
manifest._handles.find(handle => handle.id == id)  && manifest.fileName
```

File: runtime/arc.js (hunk 1 above)
Comment (shans) [reply runtime/arc.js]: done

**Pull request merged**